.github/workflows/deploy.yml

```
@@ -60,7 +60,7 @@ jobs:
         run: |
           echo "BRANCH_NAME=dev" >> $GITHUB_ENV
           if [ "${{ matrix.channel }}" == "stable" ]; then
-            echo "BRANCH_NAME=main" >> $GITHUB_ENV
+            echo "BRANCH_NAME=154-use-rcd@main" >> $GITHUB_ENV
```

```suggestion
            echo "BRANCH_NAME=main" >> $GITHUB_ENV
```

Have to revert this before we merge. Also, remember to bring the changes to the `dev` branch.